lib/matplotlib/axis.py

```diff
@@ -1320,7 +1320,13 @@ def get_minor_formatter(self):
         return self.minor.formatter
 
     def get_major_ticks(self, numticks=None):
-        """Get the tick instances; grow as necessary."""
+        """
+        Get the tick instances; grow as necessary.
+        
+        Returns
+        -------
+        list of `.Tick` objects 
```

Comment:
```suggestion
        list of `.Tick`
```
We also just say "list of str" not "list of str objects".